### PR TITLE
wrangler: remove unwanted logging from test runs

### PR DESCRIPTION
**Pull request by petebacondarwin**
- converted readonly static use of Ink to render via logger
- added console mocking to tests that need it
- add support for `console.info()` to mock console
- strip progress messages from pages upload snapshots
- remove unused console.logs from test fixture sample workers

### CODE REVIEW

File: packages/wrangler/src/__tests__/dev.test.tsx [hunk 1]
```diff
@@ -635,7 +635,7 @@ describe("wrangler dev", () => {
 		it("should run a custom build before starting `dev`", async () => {
 			writeWranglerToml({
 				build: {
-					command: `node -e "console.log('custom build'); require('fs').writeFileSync('index.js', 'export default { fetch(){ return new Response(123) } }')"`,
+					command: `node -e "4+4; require('fs').writeFileSync('index.js', 'export default { fetch(){ return new Response(123) } }')"`,
```
Comment (rozenmd) [inline packages/wrangler/src/__tests__/dev.test.tsx:638]: why the 4+4?

Comment (JacobMGEvans) [reply packages/wrangler/src/__tests__/dev.test.tsx:638]: I think its fine.

Comment (rozenmd) [reply packages/wrangler/src/__tests__/dev.test.tsx:638]: All good, was just curious

**Review by JacobMGEvans — APPROVED** [review]
This is awesome!

**Pull request merged**